lncfg/db.go

```diff
@@ -0,0 +1,85 @@
+package lncfg
+
+import (
+	"fmt"
+
+	"github.com/lightningnetwork/lnd/channeldb/kvdb"
+	"github.com/lightningnetwork/lnd/channeldb/kvdb/etcd"
+)
+
+const (
+	dbName      = "channel.db"
+	boltBackend = "bolt"
+	etcdBackend = "etcd"
+)
+
+// BoltDB holds bolt configuration.
+type BoltDB struct {
+	NoFreeListSync bool `long:"nofreelistsync" description:"If true, prevents the database from syncing its freelist to disk"`
+}
+
+// EtcdDB hold etcd configuration.
+type EtcdDB struct {
+	Host string `long:"host" description:"Etcd database host."`
+
+	User string `long:"user" description:"Etcd database user."`
+
+	Pass string `long:"pass" description:"Password for the database user."`
+
+	CollectStats bool `long:"collect_stats" description:"Wheter to collect etcd commit stats."`
```

Comment:
This also needs TLS config info for certain configurations.

Reply:
done in separate commit (can squash after review).